### PR TITLE
[IIS Access Log]: Update documentation for custom log processing

### LINKED ISSUE #8102: [IIS] Enhance the document on support for default & custom log formats
Capture the default and custom log format supported in IIS. As IIS provides flexibility on adding and removing logging fields this creates multiple different log formats.

- Update the document to capture the current supported log formats if any format is missing.
- If the log format is not accepted by Elastic by default the user can create a custom ingest pipeline to process the logs. Add a note  on custom pipeline usage in the document.

#### Change the below sentence,
`IIS integration offers certain field combinations shipped automatically into Elasticsearch using ingest pipelines. The supported formats are listed below,`
#### Similar to,
`please check the IIS log format configuration as it must match with the formats below`

### PR DESCRIPTION
**Pull request by muthu-mps**
- Enhancement

## Proposed commit message

Add note on the custom access log format processing.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.

## Author's Checklist

- [ ] 

## Related issues

- Closes #8102

### CODE REVIEW

**Review by milan-elastic — APPROVED** [review]
Looks good!

File: packages/iis/_dev/build/docs/README.md [hunk 1]
```diff
@@ -50,7 +47,8 @@ This data stream will collect and parse access IIS logs. The supported log forma
 
 The IIS ships logs with few fields by default and if the user is interested in customizing the selection, the IIS Manager provides ability to add new fields for logging.
 
-IIS integration offers certain field combinations shipped automatically into Elasticsearch using ingest pipelines. The supported formats are listed below,
+IIS integration offers certain field combinations shipped automatically into Elasticsearch using ingest pipelines. 
+Please check the IIS log format configuration as it must match with any one of the formats below,
```
Comment (devamanv) [inline packages/iis/_dev/build/docs/README.md:51]:
```suggestion
Please check the IIS log format configuration as it must match any one of the formats below
```

File: packages/iis/docs/README.md [hunk 2]
```diff
@@ -50,7 +47,8 @@ This data stream will collect and parse access IIS logs. The supported log forma
 
 The IIS ships logs with few fields by default and if the user is interested in customizing the selection, the IIS Manager provides ability to add new fields for logging.
 
-IIS integration offers certain field combinations shipped automatically into Elasticsearch using ingest pipelines. The supported formats are listed below,
+IIS integration offers certain field combinations shipped automatically into Elasticsearch using ingest pipelines. 
+Please check the IIS log format configuration as it must match with any one of the formats below,
```
Comment (devamanv) [inline packages/iis/docs/README.md:51]:
```suggestion
Please check the IIS log format configuration as it must match any one of the formats below
```

**Comment by devamanv** [comment]
Also it's good to add a small description for the Proposed commit message or just use "see title" if the title is enough. We should start following the template which says:
>This text will be pasted into the squash dialog when the change is committed and will be a long term historical record of the change to help future contributors understand the change, please help them by making it clear and comprehensive, they may be you.

**Review by devamanv — APPROVED** [review]
LGTM

**Review by shmsr — CHANGES_REQUESTED** [review]

File: packages/iis/changelog.yml [hunk 3]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.17.3"
+  changes:
+    - description: Update document for custom log format processing.
```
Comment (shmsr) [inline packages/iis/changelog.yml:4]:
```suggestion
    - description: Update documentation for custom log format processing.
```

**Review by shmsr — APPROVED** [review]
Left one comment. Otherwise, LGTM.

**Pull request merged**